src/status_im/contexts/shell/qr_reader/view.cljs

```diff
@@ -102,7 +103,9 @@
     ;; TODO: https://github.com/status-im/status-mobile/issues/18744
     nil
 
-    (wc-utils/valid-uri? scanned-text)
+    (and
+     (wc-utils/valid-uri? scanned-text)
```

Comment:
can `valid-uri?` throw? If yes, it should short-circuit on the feature flag check, so that it never reaches this function call.

Reply:
`valid-uri?` always returns a boolean, so we should be fine.